testdata/raw/emoji/en.xml

```diff
@@ -0,0 +1,42 @@
+<ldml>
```

Comment:
It's a bit strange for `en.xml` to be an example file that is only in Estonian and not English, isn't it?